### PR TITLE
added Hyperknox.World

### PR DESCRIPTION
**Pull request by Diranus**
## Type of change

- [ ] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [ ] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [ ] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [ ] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/hyperknoxworld/manifest.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  "direct_ip": "hyperknox.world",
+  "supported_languages": [
+     "de"
+  ],
+  "social": {
+    "web": "https://hyperknox.de/",
+    "web_shop": "https://hyperknox.craftingstore.net/",
+    "web_support": "https://community.hyperknox.de/ticketsystem/ticket-add/",
+    "twitter": "HyperknoxWorld",
+    "tiktok": "",
```
Comment (JakobBruening) [inline minecraft_servers/hyperknoxworld/manifest.json:13]: Do not let this empty. Add a value or remove.

File: minecraft_servers/hyperknoxworld/manifest.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    "facebook": "HyperknoxWorld/",
+    "instagram": "hyperknoxworld/",
```
Comment (JakobBruening) [inline minecraft_servers/hyperknoxworld/manifest.json:15]: Remove the `/` at the end

File: minecraft_servers/hyperknoxworld/manifest.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  "supported_languages": [
+     "de"
+  ],
+  "social": {
+    "web": "https://hyperknox.de/",
+    "web_shop": "https://hyperknox.craftingstore.net/",
+    "web_support": "https://community.hyperknox.de/ticketsystem/ticket-add/",
+    "twitter": "HyperknoxWorld",
+    "tiktok": "",
+    "facebook": "HyperknoxWorld/",
```
Comment (JakobBruening) [inline minecraft_servers/hyperknoxworld/manifest.json:14]: Remove the `/` at the end

File: minecraft_servers/hyperknoxworld/manifest.json [hunk 4]
```diff
@@ -0,0 +1,45 @@
+{
+  "server_name": "hyperknoxworld",
+  "nice_name": "Hyperknox World",
+  "direct_ip": "hyperknox.world",
+  "supported_languages": [
+     "de"
+  ],
+  "social": {
+    "web": "https://hyperknox.de/",
+    "web_shop": "https://hyperknox.craftingstore.net/",
```
Comment (JakobBruening) [inline minecraft_servers/hyperknoxworld/manifest.json:10]: We would prefer removing the `/` at the end here (not required).

File: minecraft_servers/hyperknoxworld/manifest.json [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    "web_support": "https://community.hyperknox.de/ticketsystem/ticket-add/",
```
Comment (JakobBruening) [inline minecraft_servers/hyperknoxworld/manifest.json:11]: We would prefer removing the `/` at the end here (not required).

File: minecraft_servers/hyperknoxworld/manifest.json [hunk 6]
```diff
@@ -0,0 +1,45 @@
+{
+  "server_name": "hyperknoxworld",
+  "nice_name": "Hyperknox World",
+  "direct_ip": "hyperknox.world",
+  "supported_languages": [
+     "de"
+  ],
+  "social": {
+    "web": "https://hyperknox.de/",
```
Comment (JakobBruening) [inline minecraft_servers/hyperknoxworld/manifest.json:9]: We would prefer removing the `/` at the end here (not required).

**Review by JakobBruening — CHANGES_REQUESTED** [review]
- logo@2x-fs8.png isn't a valid file name. Rename or remove.
- logo-fs8.png isn't a valid file name. Rename or remove.

**Comment by Diranus** [comment]
> _Just as an information_:
> Your server hyperknox.world **could be offline**.
> In general, we only accept pull requests from servers, **that are online**. Please change this, otherwise we cannot review your server correctly and have to deny the pull request.
> 
> If your server is currently online, then our api returned a wrong status, we will have a look at it :)
> 
> Reference: [API URL (https://api.mcsrvstat.us/2/hyperknox.world)](https://api.mcsrvstat.us/2/hyperknox.world)

not offline.

The address to join the server appears to be wrong.

It's "hyperknox.world" (without the "minecraft." prefix)

**Review by JakobBruening — APPROVED** [review]
Looks good, thank you 👍🏼

**Comment by JakobBruening** [comment]
@Diranus It's not our detection, it's from an external api.
The hostname is detected via a srv record I guess. You should check that.
Bildschirmfoto 2021-08-31 um 22 56 48


But it doesn't matter for LABY.net, because the api only needs to give server data, when there is no server-media entry.

**Pull request merged**